File: Praktikum 1/Bahasa C/PRAK110-2210817210028-Muhammad Aulia Rasyid.c
#include <stdio.h>

int main()
{
    float A , B , C , k , l;
    A = 12;
    C = 5;
    B = sqrt((pow(C,2)) + (pow(A,2)));
    printf("Diketahui:\n");
    printf("Alas = %.0f cm\n", C);
    printf("Tinggi = %.0f cm\n\n", A);
    printf("Jawab:\n");
    printf("Sisi A = %.0f cm\n", A);
    printf("Sisi B = %.0f cm\n", B);
    printf("Sisi C = %.0f cm\n", C);
    k = A + B + C;
    printf("Keliling = %.0f cm\n", k);
    l = 0.5 * C * A;
    printf("Luas = %.0f", l);
    return 0;
}

File: Praktikum 1/Bahasa Python/PRAK110-2210817210028-Muhammad Aulia Rasyid.py
import math
A = 12
C = 5
B = math.sqrt((pow(C,2)) + (pow(A,2)))
print("Diketahui:")
print("Alas = %.0f cm"% C)
print("Tinggi = %.0f cm\n"% A)
print("Jawab:")
print("Sisi A = %.0f cm"% A)
print("Sisi B = %.0f cm"% B)
print("Sisi C = %.0f cm"% C)
k = A + B + C
print("Keliling = %.0f cm"% k)
l = 0.5 * C * A
print("Luas = %.0f"% l)